### PR TITLE
Added web.dev

### PR DESCRIPTION
**Pull request by ParaBolt**
Hi, back with one more!

### CODE REVIEW

File: helpers/web.dev.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "web.dev",
```
Comment (stefanjudis) [inline helpers/web.dev.json:2]:
```suggestion
  "name": "web.dev Measure",
```

File: helpers/web.dev.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "desc": "Get the web's modern capabilities on your own sites and apps with useful guidance and analysis from web.dev",
+  "url": "https://web.dev/measure/",
+  "tags": [
+    "Site analyzers"
+  ],
+  "maintainers": [
+    "googlechrome"
```
Comment (stefanjudis) [inline helpers/web.dev.json:9]:
```suggestion
    
```

So far we're only listing maintainers when it's on GitHub and maintained by a person. :)

**Comment by ParaBolt** [comment]
Hey, I have updated the request with the changes!

**Comment by stefanjudis** [comment]
Sweet! Thank you! :)

**Pull request merged**